pycbc/population/rates_functions.py

```diff
@@ -36,12 +38,12 @@ def process_full_data(fname, rhomin, mass1, mass2, lo_tlt_mass, hi_tlt_mass):
         id_bkg = bulk['background_exc/template_id'][:]
         id_fg = bulk['foreground/template_id'][:]
 
-        bound = np.sign((mass1[id_bkg] - lo_tlt_mass) * (hi_tlt_mass - mass1[id_bkg]))
-        bound += np.sign((mass2[id_bkg] - lo_tlt_mass) * (hi_tlt_mass - mass2[id_bkg]))
-        idx_bkg = np.where(bound == 2)
-        bound = np.sign((mass1[id_fg] - lo_tlt_mass) * (hi_tlt_mass - mass1[id_fg]))
-        bound += np.sign((mass2[id_fg] - lo_tlt_mass) * (hi_tlt_mass - mass2[id_fg]))
-        idx_fg = np.where(bound == 2)
+        mchirp_bkg = mchirp_from_mass1_mass2(mass1[id_bkg], mass2[id_bkg])
+        bound = np.sign((mchirp_bkg - lo_mchirp) * (hi_mchirp - mchirp_bkg))
```

Comment:
This may give wrong results if `lo_mchirp` is actually larger than `hi_mchirp`.  To prevent confusing outputs, should put in an immediate check for this, i.e. `assert lo_mchirp < hi_mchirp`.

Reply:
applied this in pycbc_population_rates. Also fixed all dest in pycbc_population_rates